students/tony7kh/JS_HW1/index.js

```diff
@@ -0,0 +1 @@
+alert('Using second option for including JS')
```

Comment:
Consider to use or single quotes `'` neither double quotes `""` across all script files

Reply:
The same with NL

Reply:
Fix it